packages/babel-helper-create-class-features-plugin/src/index.ts

```diff
@@ -168,15 +168,18 @@ export function createClassFeaturePlugin({
 
         if (!props.length && !isDecorated) return;
 
-        let ref;
-
+        let ref = undefined;
```

Comment:
We don't need to explicitly initialize it to `undefined`, it's the default value.